puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/hammerCliKatelloNightlyRpmBuild.groovy

```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent { label 'admin && sshkey' }
+
+    option {
+        timestamps()
+        timeout(time: 30, unit: 'MINUTES')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+
+        stage ("Build Gem") {
+            steps {
+
+                dir('hammer_cli_katello') {
+
+                    git url: "https://github.com/katello/hammer-cli-katello", branch: 'master'
+                    sh "gem build hammer_cli_katello.gemspec"
+
+                }
+            }
+        }
+
+        stage ("Setup Foreman-Packaging Repository") {
+            steps {
+                git url: "https://github.com/theforeman/foreman-Packaging", branch: "rpm/develop"
+                sh "mv ./hammer_cli_katello/*.gem ./packages/katello/rubygem-hammer_cli_katello/"
+            }
+        }
+
+        stage ("Make Scratch Build") {
+            steps {
+
+                obal(
```

Comment:
I think if we take the same idea as (https://github.com/theforeman/foreman-packaging/blob/rpm/develop/package_manifest.yaml#L283) and add it to the rubygem-hammer_cli_katello entry, when the gem is archived in the step above, we could self reference this job to get the artifact and let the FetchBuilder do the rest.